### ISSUE TITLE
cpanm --installdeps . --reinstall doesn't do anything

### ISSUE THREAD

**Issue by autarch** [miyagawa/cpanminus#512]
I haven't dug into it but I'm guessing that `--reinstall` is ignored when installing from a cpanfile?

**Comment by miyagawa** [comment]
Yep. `--reinstall` only works against a module name given as arguments.

**Comment by XSven** [comment]
Ok and what is the workaround that means how can I turn off the **locally installed version check** if I am using --installdeps?
